deploy/helm/opa-operator/crds/crds.yaml

```diff
@@ -137,6 +139,82 @@ spec:
                                 - kerberosSecretClassName
                                 - ldapServer
                               type: object
+                            experimentalEntraBackend:
```

Comment:
I prefer `experimentalEntra` over `experimentalEntraBackend`, as we don't suffix `Backend` anywhere else